be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -141,6 +137,44 @@ Status BetaRowsetWriter::add_block(const vectorized::Block* block) {
     return _add_block(block, &_segment_writer);
 }
 
+Status BetaRowsetWriter::_generate_delete_bitmap(int32_t segment_id) {
+    SCOPED_RAW_TIMER(&_delete_bitmap_ns);
+    if (_context.tablet_schema->is_partial_update()) {
```

Comment:
It is better to place this conditional statement after the conditional statement on L46.
Check MoW first, then partial update.

Reply:
Thanks, updated.